### PR TITLE
do some cleanup in kube pkg

### PR DESCRIPTION
**Pull request by gy95**
modify some comment to comply with programming specifications, and simplify some codes

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ X] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[ ] Does not have any [user-facing](https://github.com/istio/istio/tree/master/releasenotes#when-to-add-release-notes) changes. This may include CLI changes, API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

File: pkg/kube/client.go [hunk 1]
```diff
@@ -1027,7 +1026,7 @@ func removeEmptyFiles(files []string) []string {
 
 func isEmptyFile(f string) bool {
 	fileInfo, err := os.Stat(f)
-	if err != nil {
+	if err != nil && !os.IsExist(err) {
```
Comment (hzxuzhonghu) [inline pkg/kube/client.go:1029]: I think for other err, the fileInfo will be nil
And caling fileInfo.Size() will panic

Comment (gy95) [reply pkg/kube/client.go:1029]: oh, yes, i thought if we need to judge the file exist, there are two conditons: one is err is nil, the other is err is isExist Error, but    the source code is only when err is not nil will return non-nil fileInfo

**Review by hzxuzhonghu — APPROVED** [review]
/ok-to-test

**Pull request merged**